components/Navbar.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { useRouter } from 'next/router';
 import MenuIcon from '@mui/icons-material/Menu';
+import LogoSVG from '../public/images/LogoSVG';
```

Comment:
Should the LogoSVG component be in the components folder, or the public/images folder?

Reply:
I think it make sense in the public/images folder but we should double check with @egoroza